analyzers/tests/SonarAnalyzer.UnitTest/PackagingTests/RuleTypeTest.cs

```diff
@@ -63,8 +72,7 @@ private static void DetectTypeChanges(ResourceManager resourceManager, IImmutabl
 
             // IMPORTANT: Rules should not be deleted without careful consideration and prior
             // deprecation. We need to notify the platform team as well.
```

Comment:
```suggestion
            // deprecation. We need to notify the product teams (SQ, SC, SL) as well.
```

Reply:
IMO this comment about notification should also be put above the `deletedRules` field - I find it's a good reminder